contracts/protocore/GenesisOriginProtocore.sol

```diff
@@ -24,18 +24,33 @@ contract GenesisOriginProtocore {
 
     /* Storage */
 
-    /** Block hash of source checkpoint */
+    /** Metachain id of the origin chain. */
+    bytes32 public genesisOriginMetachainId;
```

Comment:
`genesisMetachainId` ?

Also, contract's documentation should be updated.

Reply:
Updated.